test/testenv/verificationutils.go

```diff
@@ -489,3 +500,45 @@ func VerifySplunkServerConfSecrets(deployment *Deployment, testenvInstance *Test
 		}
 	}
 }
+
+// VerifyPVCExists verifies if PVC exist
+func VerifyPVCExists(deployment *Deployment, testenvInstance *TestEnv, ns string, pvcName string, expectedtoExist bool, verificationTimeout time.Duration) {
+	gomega.Eventually(func() bool {
+		result := false
+		pvcexists := false
+
+		pvcsList := DumpGetPvcs(testenvInstance.GetName())
+		for i := 0; i < len(pvcsList); i++ {
+			if strings.EqualFold(pvcsList[i], pvcName) {
+				pvcexists = true
+				break
+			}
+		}
+		if expectedtoExist && pvcexists || !expectedtoExist && !pvcexists {
+			result = true
+			logf.Log.Info("PVC status (exists or not) is as expected: ", "PVC", pvcName, "STATUS", pvcexists, "EXPECTED", expectedtoExist)
+		} else {
+			err := errors.New("incorrect PVC status (exists when it shouldn't or the opposite): ")
```

Comment:
This can be just an INFO Log. As test will fail in this scenario. Also instead of using logf 
`testenvInstance.Log.Info("Incorrect PVC Status", "PVC NAME", pvcName, "STATUS", pvcexists, "EXPECTED", expectedtoExist)`